packages/cursorless-engine/src/index.ts

```diff
@@ -6,3 +6,7 @@ export * from "./core/StoredTargets";
 export * from "./typings/TreeSitter";
 export * from "./cursorlessEngine";
 export * from "./api/CursorlessEngineApi";
+export * from "./CommandRunner";
+export * from "./util/getPartialTargetDescriptors";
+export * from "./util/getPrimitiveTargets";
+export * from "./util/getScopeType";
```

Comment:
A bit of a smell to have to export all these; I really think as much of command history should be engine-side

Reply:
I guess we could move the Cursorless version and the command history setting to the engine?

Reply:
yeah I'd be tempted to just move it all to the engine. No reason to arbitrarily make it harder for other ides in the future

Reply:
Agreed

Reply:
Hmm. Let's put it there for now